src/Illuminate/Queue/Failed/DatabaseFailedJobProvider.php

```diff
@@ -70,7 +70,7 @@ public function log($connection, $queue, $payload, $exception)
      */
     public function all()
     {
-        return $this->getTable()->orderBy('id', 'desc')->get()->all();
+        return $this->getTable()->get(['id'])->all();
```

Comment:
This method should return "a list of all of the failed jobs". Isn't this changing it so the docs are no longer correct? This would also break existing failed jobs providers.

Reply:
Yes. I see your point. If we all agree the current flow is optimal, I will create a new method for this.

Reply:
I think you can do  `$this->getTable()->pluck('id');` right?